packages/vscode-extension/src/commands/FileImportCommon.ts

```diff
@@ -68,18 +68,29 @@ export default abstract class FileImportCommon extends BaseAction {
 			return;
 		}
 
-		const commandArgs = {
-			paths: selectedFilesPaths,
-			...(excludeProperties === this.translationService.getMessage(ANSWERS.YES) && { excludeproperties: 'true' }),
-		};
+		const commandArgs = this.getCommandArgs(selectedFilesPaths, excludeProperties);
 
 		const commandActionPromise = this.runSuiteCloudCommand(commandArgs);
-		this.messageService.showStatusBarMessage(this.translationService.getMessage(IMPORT_FILES.IMPORTING_FILES), true, commandActionPromise);
+		let statusMessage = this.translationService.getMessage(IMPORT_FILES.IMPORTING_FILES);
+		if (commandArgs.calledfromupdate === 'true') {
+			statusMessage = this.translationService.getMessage(IMPORT_FILES.UPDATING_FILE);
+		}
+		this.messageService.showStatusBarMessage(statusMessage, true, commandActionPromise);
 		const actionResult = await commandActionPromise;
 
 		this.showOutput(actionResult);
 	}
 
+	getCommandArgs(
+		selectedFilesPaths: string[],
+		excludeProperties: string
+	): { excludeproperties?: string | undefined; paths: string[]; calledfromupdate?: string } {
+		return {
+			paths: selectedFilesPaths,
+			...(excludeProperties === this.translationService.getMessage(ANSWERS.YES) && { excludeproperties: 'true' }),
+		};
+	}
+
```

Comment:
This could be enhanced a little bit. 
I think it should be a protected method that returns `{ excludeproperties?: boolean; paths: string[]; calledfromupdate?: string }`
Then we ovewrite it from UpdateFile.ts as you already do.

And I think we should use the same approach for the statusMessage. We could create a protected getStatusMessage() that returns the `importing files...` and ovewrite it from UpdateFile.ts to return `updating file...`

Reply:
excludeproperties should be a string, since this argument is passed as argument of the command.

For the rest, I would do it as you say.